web/html/src/branding/css/base/components/buttons.suma.scss

```diff
@@ -31,7 +31,7 @@
   }
 
   // Branding hover effect
-  &:not(:disabled):not(.disabled):not(.text-muted):hover::before {
+  &:not(:disabled):not(.disabled):not(.text-muted):hover::after {
```

Comment:
I would recommend adding a comment here that it has to be `after` and why, otherwise this bug might creep in again at a later date. 🙂